payments/card/other-features.md

```diff
@@ -29,6 +29,8 @@ covered by the other subsections." %}
 {% include payment-resource.md api_resource="creditcard" documentation_section="card"
 show_status_operations=true %}
 
+{% include card-operation-paid.md %}
```

Comment:
Why is `paid` documented as a part of this PR? Shouldn't that be done in DX-1017?